src/components/common/Button/Button.tsx

```diff
@@ -10,6 +10,7 @@ interface ButtonProps {
   hero?: boolean
   modal?: boolean
   intro?: boolean
+  onClick: () => void
```

Comment:
this should be deleted as we should wrap next/link around the Button component for page navigation